contributor_docs/access.md

```diff
@@ -0,0 +1,34 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means we prioritize the needs of marginalized groups over the continued comfort of more privileged groups with p5.js.
+
+This is not the first conversation we've had about access and inclusion. We share similar intentions in our [Community Statement](../CODE_OF_CONDUCT.md), which was written at the [2015 Contributors Conference](https://p5js.org/community/contributors-conference-2015.html).
```

Comment:
Maybe reword "This is not the first conversation we've had about access and inclusion..." to "This is part of an ongoing conversation about access and inclusion within p5.js. Our intention to hold these as core values from which p5.js is built is laid out in our [Community Statement]..."

Reply:
Maybe one more statement to the effect of: "We engage in accessibility work in the context of anti-racist work, and believe it's necessary to understand the ways that factors such as disability, gender, sexuality, and class intersect with, and disproportionately affect people based on race." I don't know if that's quite it but trying to get there...

Reply:
> This is part of an ongoing conversation

@lmccart i like this framing a lot better thank you! will think about phrasing for the "we engage" statement. if anyone else has ideas for that please share! :)

Reply:
@lmccart does this line we added cover this point you had? or is there another point to make sure we cover?

> This means considering the vectors of diversity (e.g. gender, social, economic, race, ethnicity, language, ability, etc.) that can impact access/participation; and taking action to acknowledge, dismantle, and prevent barriers.

Reply:
@outofambit I want to take a crack at one more line after that paragraph that speaks to intersectionality. I think right now we are mentioning the different vectors to consider within diversity but we might recognize how these play into each other and shouldn't be treated in isolation. I can work on this in the next couple days and propose something.